README.md

```diff
@@ -110,6 +110,19 @@ HOSTNAME_COMMAND=wget -t3 -T2 -qO-  http://169.254.169.254/latest/meta-data/loca
 ```
 Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
 
+For Local, OpenShift, Kubernetes or Swarm deployments you can use the following to get the container host's IP:
+```
+HOSTNAME_COMMAND=ip -o -4 addr list eth0 | awk '{print \$4}' | cut -d/ -f1
+```
+Note: Here `$4` is escaped with `\`, which is dependent on where you use/define the environment variable.
```

Comment:
Typically I have seen people configuring docker containers via docker-compose when there are multiple parameters. Here the escaping only applies to starting things from a `sh` / `bash` shell so isn't universally applicable.

Reply:
That's the point I was trying to make, that it only needs to be escaped on the command line (I should re-word it). I actually found a better one that works for me that doesn't crumble:

`ip -o -4 addr show dev eth0 | cut -d' ' -f7 | cut -d'/' -f1`

Reply:
There's no example usage or mention of using the command line to manually `docker run` the image. The rest of the documentation assumes/describes docker-compose so this feels out of place.